mintpy/closure_phase_bias.py

```diff
@@ -8,39 +8,106 @@
-from mintpy.utils import readfile, writefile
+from mintpy.objects import ifgramStack, cluster
+from mintpy.utils import readfile, writefile, ptime, isce_utils, arg_group
 from mintpy import ifgram_inversion as ifginv
 
-
 ################################################################################
 REFERENCE = """reference:
-  Zheng, Y., et al., (2022) On Closure Phase and Systematic Bias in Multilooked SAR Interferometry, IEEE TGRS, under review (minor revision)
+  Y. Zheng, H. Fattahi, P. Agram, M. Simons and P. Rosen, "On Closure Phase and Systematic Bias in Multi-looked SAR Interferometry," in IEEE Transactions on Geoscience and Remote Sensing, doi: 10.1109/TGRS.2022.3167648.
 """
 EXAMPLE = """example:
-  closure_phase_bias.py -i inputs/ifgramStack.h5 --nl 20 --numsigma 2.5
+    closure_phase_bias.py -i inputs/ifgramStack.h5 --nl 20 --action create_mask
+    closure_phase_bias.py -i inputs/ifgramStack.h5 --nl 20 --numsigma 2.5 --action create_mask
+    closure_phase_bias.py -i inputs/ifgramStack.h5 --nl 20 --bw 10 -a quick_biasEstimate
+    closure_phase_bias.py -i inputs/ifgramStack.h5 --nl 20 --bw 10 -a biasEstimate
 """
 
 def create_parser():
-    parser = argparse.ArgumentParser(description = 'Create an indication map for closure phase bias.')
+    parser = argparse.ArgumentParser(description = 'This script deals with phase non-closure related bias, either in terms of masking, estimation or correction.',
+                                    formatter_class = argparse.RawTextHelpFormatter,
+                                    epilog=REFERENCE+'\n'+EXAMPLE)
     parser.add_argument('-i','--ifgramstack',type = str, dest = 'ifgram_stack',help = 'interferogram stack file that contains the unwrapped phases')
     parser.add_argument('--nl', dest = 'nl', type = int, default = 20, help = 'connection level that we are correcting to (or consider as no bias)')
+    parser.add_argument('--bw', dest = 'bw', type = int, default = 10, help = 'bandwidth of time-series analysis that you want to correct')
     parser.add_argument('--numsigma',dest = 'numsigma', type = float, default = 3, help = 'Threashold for phase (number of sigmas,0-infty), default to be 3 sigma of a Gaussian distribution (assumed distribution for the cumulative closure phase) with sigma = pi/sqrt(3*num_cp)')
     parser.add_argument('--epi',dest = 'episilon', type = float, default = 0.3, help = 'Threashold for amplitude (0-1), default 0.3')
     parser.add_argument('--maxMemory', dest = 'max_memory', type = float, default = 8, help = 'max memory to use in GB')
+    parser.add_argument('--noupdate_CP',dest = 'update_CP', action = 'store_false', help = 'Use when no need to compute closure phases')
     parser.add_argument('-o', dest = 'outdir', type = str, default = '.', help = 'output file directory')
+    parser.add_argument('-a','--action', dest='action', type=str, default='create_mask',
+                        choices={'create_mask', 'quick_biasEstimate', 'biasEstimate'},
```

Comment:
```suggestion
                        choices={'create_mask', 'quick_bias_estimate', 'bias_estimate'},
```

Please update its usage in the script as well.

Reply:
Updated.

Reply:
I do not see code changes. Could you double-check if you have committed/pushed the changes?

Reply:
Should be there now.